packages/eslint-plugin/src/unsupported-route-components.ts

```diff
@@ -18,28 +42,42 @@ export const unsupportedRouteComponents = createRule({
       unexpected:
         'Unexpected JSX element <{{name}}>. Only <Router>, <Route>, <Set>, <PrivateSet> and <Private> are allowed in the Routes file.',
```

Comment:
Do we need to change the error message to be more specific too?

Reply:
No I don't think so, because the problem was that it was targetting all JSX Elements.  The behaviour actually hasn't changed

Reply:
> Only <Router>, <Route>, <Set>, <PrivateSet> and <Private> are allowed in the Routes file.

It's not true though, is it? You can have `const AnotherThing = <Bazinga><p>Hello</p></Bazinga>` in your Routes file and it's still perfectly valid, right? We just want to warn against doing crazy things inside the `<Router>` element itself, not the entire file.

Reply:
Oh I see what you're saying!

Reply:
Updated!